client/src/app/(validator)/data-validation/assets/table/status.tsx

```diff
@@ -21,16 +57,143 @@ const text: Record<Status, string> = {
+                  <Typography>
+                    <code>{(error.message as TempValidation).message}</code>
+                  </Typography>
+                  <div className="rounded-xl border border-border/20 bg-background p-4">
+                    <code>{`[`}</code>
+                    {error.message &&
+                      typeof error.message === "object" &&
+                      (error.message as TempValidation).errors?.map((data) => (
+                        <Typography
+                          key={`${data.key}-${data.path}`}
+                          className="pl-8"
+                        >
+                          <code>{`{ "key": "${data.key}", "path": "${data.path}" }`}</code>
+                        </Typography>
+                      ))}
+                    <code>{`]`}</code>
+                  </div>
+                </AccordionContent>
+              </AccordionItem>
+            );
+          })}
+
+          {validations.warnings.map((warning) => {
+            return (
+              <AccordionItem
+                value={`${assetName}-${warning.validatorId}`}
+                key={`${assetName}-${warning.validatorId}`}
+                className="border-border/20 pb-2 pt-1"
+              >
+                <AccordionTrigger>
+                  <div className="flex flex-row items-center gap-4">
+                    <div className="flex h-9 w-9 items-center justify-center rounded-full bg-warning/10">
+                      <ExclamationIcon className="text-warning" />
+                    </div>
+                    <Typography>
+                      {hyphenToTitleCase(warning.validatorId)}
+                    </Typography>
+                  </div>
+                </AccordionTrigger>
+                <AccordionContent className="flex flex-col gap-2 rounded-xl border border-warning bg-warning/10 p-4">
+                  <Typography>
+                    <code>{(warning.message as TempValidation).message}</code>
+                  </Typography>
+                  <div className="rounded-xl border border-border/20 bg-background p-4">
+                    <code>{`[`}</code>
+                    {warning.message &&
+                      typeof warning.message === "object" &&
+                      (warning.message as TempValidation).warnings?.map(
+                        (data) => (
+                          <Typography
+                            key={`${data.key}-${data.path}`}
+                            className="pl-8"
+                          >
+                            <code>{`{ "key": "${data.key}", "path": "${data.path}" }`}</code>
+                          </Typography>
+                        ),
+                      )}
+
+                    <code>{`]`}</code>
+                  </div>
+                </AccordionContent>
+              </AccordionItem>
+            );
+          })}
+        </Accordion>
+        <DialogFooter className="flex flex-row items-center justify-end">
```

Comment:
I think the footer could be removed

Reply:
why ? 
then I have to use a div to put it in the right corner

Reply:
Im saying the entire footer and its content could be removed because we dont need a close button at the bottom of the dialog. there is already the 'X' icon to close the dialog in the top right. but its just a suggestion, if you would want both close buttons we can have both:)

Reply:
ahh good, yeah i'll remove it